codegen/smithy-ruby-codegen-test/model/component-test/compression.smithy

```diff
@@ -0,0 +1,35 @@
+$version: "1.0"
+namespace smithy.ruby.tests
+
+@requestCompression(
+    encodings: ["gzip"]
+)
+
+@http(method: "POST", uri: "/request_compress_operation")
+operation RequestCompressionOperation {
+    input: RequestCompressionInput
+}
+
+@http(method: "POST", uri: "/request_compress_streaming_operation")
+operation RequestCompressionStreamingOperation {
+    input: RequestCompressionStreamingInput
+}
+
+@input
+structure RequestCompressionInput {
+    @httpPayload
+    body: SomeBody
```

Comment:
I think a `Blob` shape is already defined somewhere - shouldn't need SomeBody

Reply:
Great, I think I found that one and another one for streaming `blob` - thanks!